gdk4-win32/Gir.toml

```diff
@@ -0,0 +1,93 @@
+[options]
+girs_directories = ["../gir-files"]
+library = "GdkWin32"
+version = "4.0"
+min_cfg_version = "4.0.0"
+target_path = "."
+work_mode = "normal"
+use_gi_docgen = true
+single_version_file = true
+generate_safety_asserts = true
+deprecate_by_min_version = true
+trust_return_value_nullability = true
+
```

Comment:
Maybe you need a external_libraries = ["Win32"] here?

Reply:
we don't have win32 external library.. closest would be windows-rs I guess, but not really compatible with our usage. I have just defined a crate::win32. Seems to be enough, for now. We can add a feature later for windows-rs, eventually.